gnome-extra/gnome-boxes/gnome-boxes-43.0.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+VALA_MIN_API_VERSION="0.40"
+
+inherit gnome.org gnome2-utils linux-info meson readme.gentoo-r1 vala xdg
+
+DESCRIPTION="Simple GNOME application to access remote or virtual systems"
+HOMEPAGE="https://wiki.gnome.org/Apps/Boxes"
+
+LICENSE="LGPL-2+ CC-BY-2.0"
+SLOT="0"
+
+KEYWORDS="~amd64"
+
+# FIXME: qemu probably needs to depend on spice[smartcard] directly with USE=spice
+# FIXME: Check over libvirt USE=libvirtd,qemu and the smartcard/usbredir requirements
+# Technically vala itself still ships a libsoup vapi, but that may change, and it should be better to use the .vapi from the same libsoup version
+# gtk-vnc raised due to missing vala bindings in earlier ebuilds
+DEPEND="
+	>=app-arch/libarchive-3.0.0:=
+	>=dev-libs/glib-2.52:2
+	>=x11-libs/gtk+-3.24.1:3
+	>=net-libs/gtk-vnc-0.8.0-r1[gtk3(+)]
+	>=gui-libs/libhandy-1.5.0:1=
+	>=sys-libs/libosinfo-1.10.0
+	app-crypt/libsecret
+	>=net-libs/libsoup-3.0.8:3.0
```

Comment:
No need for the `>= 3.0.8` since the dependency in `meson.build` is just `dependency ('libsoup-3.0')`.